app/templates/components/ui-table/cell/cell-simple-sessions.hbs

```diff
@@ -0,0 +1,5 @@
+<div class="">
+  {{#each record as |session|}}
```

Comment:
@niranjan94 But when I try to put any random dummy content in this template also, the error is still there.